scripts/lintcommit.lua

```diff
@@ -0,0 +1,69 @@
+-- Get length of table
+function get_table_length(table)
+  local length = 0
+  for n in pairs(table) do 
+    length = length + 1 
+  end
+  return length
+end
+
+-- Split a string
+function string:split(delimiter)
```

Comment:
`vim.gsplit()` or `vim.fn.split()` ?

Reply:
OK, I'm beginning to see a pattern here :). I'll check in `lua.txt` for the function I'm looking for before I copy code from online.